tensorflow_probability/python/mcmc/dual_averaging_step_size_adaptation.py

```diff
@@ -0,0 +1,299 @@
+"""DualAveragingStepSizeAdaptation TransitionKernel."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import collections
+
+import numpy as np
+import tensorflow as tf
```

Comment:
Would it work if you do:
`import tensorflow.compat.v2 as tf`

You'll need to update downstream things, eg, s/tf.compat.v1/tf/ etc.

If you can do this it will make the review process a lot easier. We use a backend conversion script to ensire TF2 compatibility and it can be a very nitpick script.